aws/rust-runtime/aws-sigv4/Cargo.toml

```diff
@@ -1,28 +1,28 @@
 [package]
 name = "aws-sigv4"
 version = "0.1.0"
-authors = ["David Barsky <me@davidbarsky.com>"]
+authors = ["David Barsky <me@davidbarsky.com>", "AWS Rust SDK Team <aws-sdk-rust@amazon.com>"]
 edition = "2018"
-exclude = [
-    "aws-sig-v4-test-suite/*"
-]
+exclude = ["aws-sig-v4-test-suite/*"]
 license = "MIT OR Apache-2.0"
-description = "An AWS SigV4 request signer."
-repository = "https://github.com/davidbarsky/sigv4"
-homepage = "https://github.com/davidbarsky/sigv4"
-documentation = "https://docs.rs/aws-sigv4"
+description = "AWS SigV4 signer"
+
+[features]
+sign-http = ["http", "http-body", "percent-encoding", "form_urlencoded"]
+sign-eventstream = ["smithy-eventstream"]
+default = ["sign-http", "sign-eventstream"]
 
 [dependencies]
-http = "0.2"
-http-body = "0.4"
-ring = "0.16"
-serde = { version = "1", features = ["derive"] }
-serde_urlencoded = "0.7"
-bytes = "1"
-hex = "0.4"
 chrono = { version = "0.4", default-features = false, features = ["clock", "std"] }
-percent-encoding = "2.1"
+form_urlencoded = { version = "1.0", optional = true }
+hex = "0.4"
+http = { version = "0.2", optional = true }
+http-body = { version = "0.4", optional = true }
+percent-encoding = { version = "2.1", optional = true }
+ring = "0.16"
+smithy-eventstream = { path = "../../../rust-runtime/smithy-eventstream", optional = true }
 
 [dev-dependencies]
+bytes = "1"
```

Comment:
unless I'm missing something `bytes` needs to be an optional dependency that `sign-http` picks up because it's used in `SignableBody`